### PR TITLE
Removing docker jmx FAQ.

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Removes docker JMX FAQ and merge its content to:

* JMX troubleshooting page
* Main Datadog-JMX integration page

### Motivation
better doc

### Preview link

* [Preview](https://docs-staging.datadoghq.com/gus/docker-jmx-2.0/integrations/faq/troubleshooting-jmx-integrations/?tab=dockeragent#agent-troubleshooting)

### CODE REVIEW

**Comment by l0k0ms** [comment]
This needs to be merged after: https://github.com/DataDog/dogweb/pull/37129

File: content/en/integrations/faq/troubleshooting-jmx-integrations.md [hunk 1]
```diff
@@ -120,20 +120,113 @@ Note: the location to the JRE tools.jar (`/usr/lib/jvm/java-8-oracle/lib/tools.j
...
+
 {{% /tab %}}
 {{< /tabs >}}
 
-## SSL troubleshooting
+## FAQs
+
+### The 350 metric limit
+
+Due to the nature of these integrations, it is possible to submit an extremely high number of metrics directly to Datadog, note that Datadog sets the limit at maximum 350 metrics.
```
Comment (apigirl) [inline content/en/integrations/faq/troubleshooting-jmx-integrations.md]:
```suggestion
Datadog accepts a maximum of 350 metrics.
```

File: content/en/integrations/faq/troubleshooting-jmx-integrations.md [hunk 2]
```diff
@@ -120,20 +120,113 @@ Note: the location to the JRE tools.jar (`/usr/lib/jvm/java-8-oracle/lib/tools.j
...
+It is recommended to create filters to refine what metrics are collected but if you believe you need more than 350 metrics, contact [Datadog support][3].
```
Comment (apigirl) [inline content/en/integrations/faq/troubleshooting-jmx-integrations.md]:
```suggestion
A best practice is to limit your metrics to less than 350 by creating filters to refine those metrics collected, but if you need more than 350 metrics, contact [Datadog support][3].
```

File: content/en/integrations/faq/troubleshooting-jmx-integrations.md [hunk 3]
```diff
@@ -120,20 +120,113 @@ Note: the location to the JRE tools.jar (`/usr/lib/jvm/java-8-oracle/lib/tools.j
...
+                               # is used to tag the metrics pulled from that instance
+  ```
+
+* [Restart the Agent][4].
+
+### Monitoring Tomcat with JMX Remote Lifecycle Listener enabled
+
+The following instructions work on Agent v5.6.0+.
+
+If you're using Tomcat with JMX Remote Lifecycle Listener enabled (see the [Tomcat documentation][5] for more information), JMXFetch needs additional setup to be able to connect to your Tomcat application.
```
Comment (apigirl) [inline content/en/integrations/faq/troubleshooting-jmx-integrations.md]:
```suggestion
If you're using Tomcat with JMX Remote Lifecycle Listener enabled (see the [Tomcat documentation][5] for more information), JMXFetch needs an additional setup to be able to connect to your Tomcat application.
```

**Pull request merged**